src/dpq2/conv/time.d

```diff
@@ -88,16 +104,24 @@ struct TimeStampWithoutTZ
     {
         return TimeStampWithoutTZ(DateTime.min, Duration.zero);
     }
+
+    SysTime opCast()
+    {
+        return SysTime(dateTime, fracSec, UTC());
+    }
 }
 
+package enum POSTGRES_EPOCH_DATE = Date(2000, 1, 1);
```

Comment:
Already available as POSTGRES_EPOCH_JDATE

Looks like one of these enums can be removed

Reply:
I've kept the added `POSTGRES_EPOCH_DATE` but removed some now unused constants (POSTGRES_EPOCH_JDATE was there 2 times).
As I commented before, I think that a conversions can be simplified a lot using the Phobos library. But I'll look at it probably after this PR as it is already complex enough.